proposals/csharp-9.0/records.md

```diff
@@ -197,6 +197,118 @@ If the containing record is abstract, the synthesized clone method is also abstr
 If the "clone" method is not abstract, it returns the result of a call to a copy constructor. 
 
 
+### Printing members: PrintMembers and ToString
+
+If the record is derived from `object`, the record includes a synthesized method equivalent to a method declared as follows:
+```C#
+protected virtual void PrintMembers(System.StringBuilder builder, bool includeSeparator);
+```
+The method is `virtual` and `protected`.
+It is an error if the method is declared explicitly.
```

Comment:
Another reason to make it an unspeakable name. It removes this entire scenario.

Reply:
Updated to use unspeakable "print" method.